src/singletons/Settings.hpp

```diff
@@ -79,7 +79,7 @@ class Settings : public ABSettings, public ConcurrentSettings
     BoolSetting hideModerationActions = {
         "/appearance/messages/hideModerationActions", false};
     BoolSetting colorizeNicknames = {"/appearance/messages/colorizeNicknames",
```

Comment:
This is going to cause a lot of confusion to existing Chatterino users, it should at least be reflected in the changelog.